README.md

```diff
@@ -109,3 +109,15 @@ Hides the annotation if showing. This is not animated.
 
 #### remove()
 Unlinks the annotation from the element. 
+
+
+
+
+## Wrappers
+
+Rough Notation has wrappers for multiple libraries and frameworks
+
+[React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
```

Comment:
This is a bit hard to read, what about using a bulleted list?

- [React Rough Notation](https://github.com/linkstrifer/react-rough-notation)
- [Svelte Rough Notation](https://github.com/dimfeld/svelte-rough-notation)
It is easier to scan it at a glance.